pkgs/tools/admin/awscli2/default.nix

```diff
@@ -67,6 +67,8 @@ with py.pkgs; buildPythonApplication rec {
   ];
 
   postInstall = ''
+    mkdir -p $out/autocomplete/data
+    python3 scripts/gen-ac-index --index-location $out/autocomplete/data/ac.index
```

Comment:
`${python.interpreter}`

Reply:
Regarding completions, see also a7cb88c3def629f4d950d819254c27c1b328b2ac. Maybe relevant.

Reply:
Thanks for the review, let me know if the new changes are ok for you.